src/search/landmarks/exploration.cc

```diff
@@ -83,8 +83,8 @@ void Exploration::build_unary_operators(const OperatorProxy &op) {
             precondition.push_back(&propositions[precondition_fact.var]
                                    [precondition_fact.value]);
 
-        FactProxy effect_fact = effect.get_fact();
-        Proposition *effect_proposition = &propositions[effect_fact.get_variable().get_id()][effect_fact.get_value()];
+        const auto &[var, value] = effect.get_fact().get_pair();
```

Comment:
I find it weird to use const references here, since we usually just copy ints; but I'm also not familiar with the new syntax. Is there a specific reason? (The comment applies to several other places as well.)